espanol/noticias/anteojos/anteojos.md

```diff
@@ -1,7 +1,7 @@
-Una vez que mencione que necessita anteojos el periodista dice que nunca ha tenido problema sin ellos.
+Una vez que mencione que necesita anteojos el periodista dice que nunca ha tenido problema sin ellos, ya que su defecto de visión es muy leve
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.